include/graylog_logger/GraylogConnection.hpp

```diff
@@ -1,10 +1,11 @@
-//
-//  GraylogConnection.hpp
-//  dm-graylog-logger
-//
-//  Created by Jonas Nilsson on 2016-12-29.
-//  Copyright © 2016 European Spallation Source. All rights reserved.
-//
+/* Copyright (C) 2018 European Spallation Source, ERIC. See LICENSE file */
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief Header file of the networking code.
```

Comment:
This is very minor, but for the sake of standardisation, the wording could be similar for the different header files (other examples from above: "Header file console interface.",  "Header of file logging interface.")

Reply:
Done (ish).